src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -29,15 +30,15 @@ import { GiftTicketModal } from "../../organisms/GiftTicketModal/GiftTicketModal
 import PlayaTime from "../PlayaTime";
 import PlayaAddress from "../PlayaAddress";
 import { venueInsideUrl } from "utils/url";
-import { useVenueId } from "hooks/useVenueId";
+import { VenueTemplate } from "types/VenueTemplate";
 
 interface PropsType {
   redirectionUrl?: string;
 }
 
 const NavBar: React.FunctionComponent<PropsType> = ({ redirectionUrl }) => {
   const { user, profile } = useUser();
-  const venueId = useVenueId();
+  const { venueId } = useParams();
```

Comment:
you might find this doesn't work, you should test very carefully. I was seeing venueId as `undefined` in nested routing contexts such as routes in the `CampRouter`, so I suspect you should use the `useVenieId` hook.